app/controllers/claims/schools/claims_controller.rb

```diff
@@ -52,6 +52,7 @@ def submit
       claim_params: { status: :submitted, submitted_at: Time.current },
     )
 
+    User::ClaimConfirmation.call(user: current_user, claim: @claim, link_to_claim: claims_school_claim_url(id: @claim.id))
```

Comment:
Might be worth considering moving this to an `after_save` in the `Claim` model, so that when the `submitted_at` changes from `nil` it sends a confirmation. 
This will prevent spamming emails when someone hits the submit button too many times.

Reply:
Sorry to jump in here, but I'd caution against performing real-world side effects inside Active Record callbacks. It's usually not desirable. Instead, a service object is probably the more sensible place for it.

As per our [documented convention](https://github.com/DFE-Digital/itt-mentor-services/blob/main/docs/conventions/service_objects.md), we use service objects to perform data manipulation and associated side effects.

In this case, the data manipulation is _updating the claim record_ and the side effect is _sending a confirmation email_.

@gms-gs my suggestion would be that the user confirmation email is triggered from within the `Claims::Submit` service. Given its name, my understanding is that it's the service which handles the submission of claims, and this is a legitimate & desired side effect of that action.

What do you think?